src/OrchardCore/Orchard.ContentManagement/DefaultContentManager.cs

```diff
@@ -371,6 +371,17 @@ public void Create(ContentItem contentItem, VersionOptions options)
             }
         }
 
+        public void Update(ContentItem contentItem)
+        {
+            var context = new UpdateContentContext(contentItem);
+
+            Handlers.Invoke(handler => handler.Updating(context), _logger);
+            Handlers.Reverse().Invoke(handler => handler.Updated(context), _logger);
+
+            _session.Save(contentItem);
```

Comment:
Good to call `_session.Save()` after handlers, in case there are doing some queries.